### DISCUSSION TITLE
blog/how-to-keep-up-to-date-with-the-latest-r-news/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#100] [General]
# blog/how-to-keep-up-to-date-with-the-latest-r-news/

R is evolving rapidly and new functions are created everyday. See how to keep up to date with the latest R news via several twitter accounts and newsletters.

https://statsandr.com/blog/how-to-keep-up-to-date-with-the-latest-r-news/

**Comment by changzhenlin** [comment]
234

**Reply by AntoineSoetewey** [reply]
Any question @changzhenlin ?

**Reply by changzhenlin** [reply]
sorry to bother you , i`m just test your giscus. because my giscus api "https://giscus.app/api/discussions......." always 404.  🥲

**Reply by AntoineSoetewey** [reply]
Ok no problem!
